storage/pebble/registers_test.go

```diff
@@ -117,14 +165,60 @@ func Test_PayloadStorage_Versioning(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, expectedValue1ge3, value1)
 
-	value1, err = s.Get(height3+1, key1)
-	require.NoError(t, err)
-	require.Equal(t, expectedValue1ge3, value1)
+	// out of range
+	_, err = s.Get(height3+1, key1)
+	require.ErrorIs(t, err, storage.ErrHeightNotIndexed)
```

Comment:
we should also cover the case where the queried height is below the first height